web/templates/web/studies-list.html

```diff
@@ -48,7 +48,7 @@
                 <a class="text-decoration-none link-dark p-3"
                    href="{% url 'web:study-detail' uuid=study.uuid %}">
                     <img src="{{ study.image.url }}"
-                         class="img-fluid mx-auto d-block"
+                         class="w-100 mw-100 h-auto mx-auto d-block"
                          alt="{{ study.name }}"/>
```

Comment:
Are both width (w-100) and max-width (mw-100) required here?

Reply:
Ah, good catch! Nope `mx-100` isn't needed here - I'll remove it.